package.json

```
@@ -11,7 +11,8 @@
     "prepublishOnly": "npm run build",
     "bundle": "cd tools/bundler && npm i && npm run bundle",
     "startNewVersion": "newVersion=$npm_config_new_version node scripts/add-new-version.js",
-    "lint": "echo 'No linter integrated yet'",
+    "lint": "eslint --max-warnings 0 --config .eslintrc .",
+    "lint:fix": "eslint --max-warnings 0 --config . --fix",
```

```suggestion
    "lint:fix": "eslint --max-warnings 0 --config .eslintrc . --fix",
```
This should be as given [here](https://github.com/asyncapi/parser-js/blob/2671f1aeb4dd492c07b1ecdcd67d49cb33110008/package.json#L36-L37)